generators/guestcontainer/index.js

```diff
@@ -62,6 +62,11 @@ module.exports = generators.Base.extend({
                     when: !this._isOptionSet('instanceCount')
                 },
 
+                {
+                    name: 'portMap',
+                    message: 'Enter the container-host mapping of the format container_port:host_port or press enter if not needed):',
```

Comment:
Enter the container host mapping in the following format, container_port:host_port